backend/backend/settings.py

```diff
@@ -190,3 +191,15 @@
 
 def is_prod_mode():
     return DEBUG is False
+
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+
+sentry_sdk.init(
+    dsn="https://d5f3cb06dec94abe96e16bf57cd0c812@o417831.ingest.sentry.io/5319435",
```

Comment:
Looks like a secret, should go in .env

Reply:
Even .env.prod (sentry is the module to track python errors in production)